pkg/tenant/config/config.go

```diff
@@ -71,6 +73,12 @@ type EnsignConfig struct {
 	PoolPath string `split_words:"true"`
 }
 
+// Configures an SDK connection to Ensign for pub/sub.
+type SDKConfig struct {
+	Ensign  ensign.Options
+	TopicID string `split_words:"true"`
```

Comment:
Is this too tied to the SDK?

Reply:
Does the `ensign.Options` have `envconfig` variables?

Reply:
I think this is ok for now.